### DISCUSSION TITLE
Attach base64 image to Allure Reports

### DISCUSSION THREAD

**Discussion by nnson0310** [allure-framework/allure2#1718] [Questions & Support] [answered]
I am building a selenium framework (Java + Junit5)
I can easily attach screenshot to Allure Report with following code:
`File screenshotAsFile = ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);`
`Allure.addAttachment(screenshotName, FileUtils.openInputStream(screenshotAsFile));`

But I do not know how to add Base64 image to Allure Reports. I try the following way but it did not work as expected.
It just render a plain text. 

 `String screenshotAsBase64 = "<img src='data:image/png;base64," + ((TakesScreenshot) driver).getScreenshotAs(OutputType.BASE64) + "' style='width: 120px; height: 120px' />";`
`Allure.addAttachment(screenshotName,  screenshotAsBase64);`

Please help me to make Allure Reports recognize that I want to render my screenshot as Base64 Type not as a plain text. Thank you

**Comment by baev** [comment, marked as answer]
You need to specify the correct content type. In your case, you are adding `html` content, so you need to use `text/html`
